app/renderers/hyrax/renderers/attribute_renderer.rb

```diff
@@ -33,6 +33,20 @@ def render
         markup.html_safe
       end
 
+      # Draw the dl row for the attribute
+      def render_dl_row
+        markup = ''
+
+        return markup if values.blank? && !options[:include_empty]
+        markup << %(<dt>#{label}</dt>\n<dd><ul class='tabular'>)
```

Comment:
I don't believe that an unordered list is permissible inside of the `<dd>` and am unsure of the context of its use. Can you use multiple `<dd>` instead? For example, single term and multiple descriptions: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#Examples